app/model/exactTarget/package.scala

```diff
@@ -0,0 +1,29 @@
+package model
+
+package object exactTarget {
+  trait DataExtensionColumn
+
+  case object `SubscriberKey` extends DataExtensionColumn
```

Comment:
I think I'd have been okay with leaving these as simple string constants - you only have to change them back into strings again at https://github.com/guardian/subscriptions-frontend/blob/91699a3/app/services/ExactTargetService.scala#L20

It's kind of nice to have strongly-typed keys, but then relying on the `toString` method of case objects to do the right thing is a bit stressful.